markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 1 Line 5_
```markdown
> ## This is a changed header.
> 
> 1.   This is the first list **element**.
> 1.   A new item added to the list
> 1.   This is the `third list` __item__.
```
Second review comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
response to second review comment